docs/guides/01-index.md

```diff
@@ -1,11 +1,25 @@
 ## Guides
 
-* [Introduction](https://carto-python.readthedocs.io/en/latest/index.html){:target="_blank"}
-* [Quickstart](https://carto-python.readthedocs.io/en/latest/quickstart.html){:target="_blank"}
-* [General Concepts](https://carto-python.readthedocs.io/en/latest/general_concepts.html){:target="_blank"}
-* [Installation](https://carto-python.readthedocs.io/en/latest/installation.html){:target="_blank"}
-* [Authentication](https://carto-python.readthedocs.io/en/latest/authentication.html){:target="_blank"}
-* [SQL Api](https://carto-python.readthedocs.io/en/latest/sql_api.html){:target="_blank"}
-* [Import Api](https://carto-python.readthedocs.io/en/latest/import_api.html){:target="_blank"}
-* [Maps Api](https://carto-python.readthedocs.io/en/latest/maps_api.html){:target="_blank"}
-* [Examples](https://carto-python.readthedocs.io/en/latest/examples.html){:target="_blank"}
\ No newline at end of file
+This section contains documentation on how to use the different _carto-python_ APIs.
+
+_carto-python_ is a full, backwards incompatible rewrite of the deprecated _cartodb-python_ SDK. Since the initial rewrite, _carto-python_ has been loaded with a lot of new features, not present in old _cartodb-python_.
+
+_carto-python_ is a Python library to consume the CARTO APIs. You can integrate _carto-python_ into your Python projects to:
+
+* Import data from files, URLs or external databases to your user account or organization
+* Execute SQL queries and get the results
+* Run batch SQL jobs
+* Create and instantiate named and anonymous maps
+* Create, update, get, delete and list datasets, users, maps…
+etc.
+
+All the guides are hosted on [ReadTheDocs](https://carto-python.readthedocs.io/en/latest){:target="_blank"}:
+
+* [Quickstart](https://_carto-python_.readthedocs.io/en/latest/quickstart.html){:target="_blank"}
```

Comment:
Looks like the links are not correct 🤔 

Shouldn't they be `https://carto-python.readthedocs.io...`

Reply:
Ouch! I applied a replacement and didn't notice, good catch!